TechPreview for Knative-Kafka-broker docs (#39408)
openshift/openshift-docs: serverless/knative_eventing/serverless-kafka.adoc

        bootstrapServers: <bootstrap_servers> <2>
    source:
        enabled: true <3>
    broker:
        enabled: true <4>
        defaultConfig:
            bootstrapServers: <bootstrap_servers> <5>
            numPartitions: <num_partitions> <6>
            replicationFactor: <replication_factor> <7>
----
<1> Enables developers to use the `KafkaChannel` channel type in the cluster.
<2> A comma-separated list of bootstrap servers from your AMQ Streams cluster.
<3> Enables developers to use the `KafkaSource` event source type in the cluster.

> <4> Enables developers to use the Knative Kafka `Broker` implementation in the cluster.

Reviewer: ```suggestion
<4> Enables developers to use the Knative Kafka broker implementation in the cluster.
```

> <5> A comma-separated list of bootstrap servers from your AMQ Streams cluster.

Reviewer: ```suggestion
<5> A comma-separated list of bootstrap servers from your Red Hat AMQ Streams cluster.
```

> <6> Defines the number of `partitions` of the Kafka Topics, backed by the `Broker` objects. The default is `10`.

Reviewer: ```suggestion
<6> Defines the number of partitions of the Kafka topics, backed by the `Broker` objects. The default is `10`.
```

> <7> Defines the `replication factor` of the Kafka Topics, backed by the `Broker` objects. The default is `3`. 

Reviewer: ```suggestion
<7> Defines the replication factor of the Kafka topics, backed by the `Broker` objects. The default is `3`. 
```

> [NOTE]
> ====
> The value provided for `replicationFactor` must match the number of nodes of your AMQ Streams cluster.
> ====

Reviewer: ```suggestion
[NOTE]
====
The `replicationFactor` value must match the number of nodes of your Red Hat AMQ Streams cluster.
====
```

// Install Kafka
include::modules/serverless-install-kafka-odc.adoc[leveloffset=+2]

[id="serverless-kafka-broker-link"]

> == Using Kafka Broker

Reviewer: ```suggestion
== Using a Kafka broker
```

> Create a xref:../../serverless/knative_eventing/serverless-using-brokers.adoc#serverless-using-brokers-creating-kafka-brokers[Kafka Broker].

Reviewer: ```suggestion
Create a xref:../../serverless/knative_eventing/serverless-using-brokers.adoc#serverless-using-brokers-creating-kafka-brokers[Kafka broker].
```
